obs-package-status/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM opensuse/tumbleweed
+
+RUN zypper -n in osc ruby2.6 ruby2.6-devel git gcc make autoconf zlib-devel libxml2-devel libxslt-devel
+RUN gem install bundler
+RUN bundle.ruby2.6 config build.nokogiri --use-system-libraries
+COPY .  /home/package-bot/obs-package-status
+RUN cd /home/package-bot/obs-package-status && bundler.ruby2.6 install
```

Comment:
nitpick: If you move the workdir call up, it's a bit more clear.

Reply:
to where exactly?

Reply:
```suggestion
WORKDIR /home/package-bot/obs-package-status
RUN bundler.ruby2.6 install
```